### ISSUE TITLE
Evaluate Chefile syntax and capability (coming for che 4.8/4.9)

### ISSUE THREAD

**Issue by TylerJewell** [dtoebe/new-che-electron#6]
We have a new syntax that will make it tremendously easier to start a workspace on a Che server on any OS.  It is a bit like Vagrant - where it will convert a directory into a VM, but in this case we convert it into a workspace with a debugger, a set of commands, projects --- all running on a Che server. And the che server will be started in the background by the Chefile utility if one is not available.

This syntax may make it a lot easier to deal with some of the starting / stopping of Che's for the electron application.  We would like to get you a private demo sometime when you are free.  We need about 30 minutes and we can get your feedback on the work.

https://github.com/eclipse/che/pull/2048
https://github.com/eclipse/che/issues/1895

**Comment by dtoebe** [comment]
Absolutely. What time next week would be good? And do you guys record your Tuesday Hangouts? Unfortunately it goes on right when I am heading to work.

**Comment by dtoebe** [comment]
Last night I have started to build a Che Electron stack. first of all to get me antiquated with different elements of Che, and so I can honestly use Che easily to build this project.  Then I will write up a quick tut on how to build Electron apps with Che.

**Comment by TylerJewell** [comment]
We do not record the Tuesday sessions with Hangout. But they are pretty tedious discussions - it's a lot of coordination around upcoming release.  We tend to defer all architectural + technology stuff to be in Github or some other online discussion.

How about Wednesday 7am pacific for the demo?

**Comment by dtoebe** [comment]
No sorry 7a I am heading to work any time after 8a PST next week.

**Comment by TylerJewell** [comment]
Ok - let's try 8am pacific.  The engineer who I'd like to have do the demo is in France, so I will ask him to stay online a bit later.

**Comment by dtoebe** [comment]
Ok thanks
